frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -72,10 +75,16 @@ export function renderTopology({
   );
 }
 
-export const TopologyPage: React.FC<TopologyPageProps> = ({ match }) => {
+export const TopologyPageContext: React.FC<TopologyPageProps> = observer(({ match }) => {
   const queryParams = useQueryParams();
   const searchParams = queryParams.get('searchQuery');
   const namespace = match.params.name;
+  const dataModelContext = React.useContext<ExtensibleModel>(ModelContext);
+  const emptyModel =
+    dataModelContext?.isEmptyModel ??
+    (match.params.testEmptyModelValue === undefined
+      ? true
+      : match.params.testEmptyModelValue === 'true');
```

Comment:
@jeff-phillips-18  is it needed only for testing purpose? if yes,  then can we mock the useContext to return true/false in the test cases, WDYT?
eg:
 `spyOn(React, 'useContext').and.returnValue({ isEmptyModel: true });`